tests/fixtures/file_1.yaml

```diff
@@ -0,0 +1,17 @@
+common:
```

Comment:
same with files, don't replace old files - just add new ones with `nested` in name